docs/error-codes.md

```diff
@@ -55,15 +55,15 @@ Here is a summary of all the error codes thrown by the contract.
 | 308 | `internal_liquidity_below_zero_err` | Liquidity went below zero. |
 | 309 | `internal_309` | Thrown when `(p.dx - r.dx)` is not nat. |
 | 310 | `internal_insufficient_balance_err` | Contract does not have enough liquidity to execute the swap. |
-| 311 | `internal_311` | Thrown when `s.i_c >= key.hi.i` and `(s.fee_growth.x - tick_hi.fee_growth_outside.x)` (or `y`) is not nat. |
```

Comment:
_commenting here for lack of a better place_

It seems that there are a few mentions of `i_c` in comments in `types.mligo` left, which I think would better be updated too.

Reply:
I think they are fine since they appear only in documentation, and using math notation in docs seems fine and even more concise.

Although, @dcastro has raised the same concern...

Reply:
Yeah, I think it's best to use the same names throughout the entire codebase. Otherwise, readers who haven't read the paper won't know what `i_c` refers to.

On a related note, I would add a note to each _field declaration_ mentioning what the field is called in the paper, so that a reader who's read the paper can make the connection between the two, e.g.:

Reply:
(it's not a strongly held opinion tho, just my 2 cents :sweat_smile: )

Reply:
I created [TCFMM-32](https://issues.serokell.io/issue/TCFMM-32) for further discussion then. This PR is already large and reviewed once, probably I should merge it now.